docs/customizing_dashboards.md

```diff
@@ -158,6 +158,17 @@ end
 
 [define your own]: /adding_custom_field_types
 
+To change dashboard name in sidebar menu, sub-header and search string simply use default ActiveRecord i18n translations for models:
```

Comment:
I think we need some "the"s in the start here:

> To change _the_ Dashboard name in _the_ sidebar

And would you mind removing "simply"? Whilst in this case it's true, I like to avoid words in our docs like "simply", "easily", etc as this is often not true for everyone.